lib/pacemaker/pcmk_output.c

```diff
@@ -145,15 +145,15 @@ rsc_is_colocated_with_list_xml(pcmk__output_t *out, va_list args) {
         rsc_colocation_t *cons = (rsc_colocation_t *) lpc->data;
 
         if (pcmk_is_set(cons->rsc_rh->flags, pe_rsc_allocating)) {
-            PCMK__OUTPUT_LIST_HEADER(out, FALSE, rc, "Colocations");
+            PCMK__OUTPUT_LIST_HEADER(out, FALSE, rc, "rsc-is-colocated-with");
```

Comment:
In XML all the colocations could be together in a single list, since rsc/with-rsc will indicate the direction of the constraint. But for practical reasons if it's easier to separate the two lists, that's fine too

Reply:
It might be reasonable to just flatten the whole structure for XML, putting all the recursive results into a single flat list.